release_notes/other-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="other-release-notes"]
+= Other release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: other-release-notes
+
+Release notes for other related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="4,7a",options="header"]
+|===
+|Component or product |Release note documentation
+
+|Compliance Operator
+|xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator release notes] in "Security and compliance"
```

Comment:
Is it necessary to say the name of the book that the release notes are in? Just seems a little cluttered with it in there.

```suggestion
|xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator release notes]
```

Reply:
Might even just simplify it further by making all the link texts be "Release notes". I was trying to match the target document title exactly, but that's probably A) not particularly valuable and B) could slightly change on us over time anyway)